diff --git a/web/viewer.js b/web/viewer.js
index ff816fc..96fa609 100644
--- a/web/viewer.js
+++ b/web/viewer.js
@@ -2200,9 +2200,6 @@ let validateFileURL;
         protocol
       } = new URL(file, window.location.href);
 
-      if (origin !== viewerOrigin && protocol !== "blob:") {
-        throw new Error("file origin does not match viewer's");
-      }
     } catch (ex) {
       PDFViewerApplication.l10n.get("loading_error").then(msg => {
         PDFViewerApplication._documentError(msg, {
@@ -15114,4 +15111,4 @@ if (document.readyState === "interactive" || document.readyState === "complete")
 
 /******/ })()
 ;
-//# sourceMappingURL=viewer.js.map
\ No newline at end of file
+//# sourceMappingURL=viewer.js.map
