diff --git a/web/viewer.js b/web/viewer.js
index d899023..6165f90 100644
--- a/web/viewer.js
+++ b/web/viewer.js
@@ -1935,9 +1935,6 @@ let validateFileURL;
         protocol
       } = new URL(file, window.location.href);
 
-      if (origin !== viewerOrigin && protocol !== "blob:") {
-        throw new Error("file origin does not match viewer's");
-      }
     } catch (ex) {
       const message = ex && ex.message;
       PDFViewerApplication.l10n.get("loading_error", null, "An error occurred while loading the PDF.").then(loadingErrorMessage => {
@@ -13897,4 +13894,4 @@ _app.PDFPrintServiceFactory.instance = {
 
 /***/ })
 /******/ ]);
-//# sourceMappingURL=viewer.js.map
\ No newline at end of file
+//# sourceMappingURL=viewer.js.map
